### ISSUE TITLE
[bug] Getting ssl errors for conan.bintray.com

### ISSUE THREAD

**Issue by asanka424** [conan-io/conan#10025]
### Environment Details (include every applicable attribute)
  * Operating System+version: Windows
  * Compiler+version: Visual studio 2017
  * Conan version: 1.33.0
  * Python version: 3.7

### Steps to reproduce (Include if Applicable)
I started getting this SSL error today
ERROR: HTTPSConnectionPool(host='conan.bintray.com', port=443): Max retries exceeded with url: /v1/ping (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)'),))

I updated my certs with 
conan config install https://github.com/conan-io/conanclientcert.git

but no change. 

Can someone help me?

**Comment by memsharded** [comment]
Hi @asanka424 

This is expected. bintray is closed today in a brownout. Please read: https://blog.conan.io/2021/10/28/conancenter-bintray-remote-eol.html

Also announced in social, like: https://twitter.com/conan_io/status/1460534314379223041?s=20


TL;DR: conan.bintray.com is being closed. You need to upgrade to the new URL in center.conan.io asap.

**Comment by asanka424** [comment]
I see. got it.  I will update

**Comment by asanka424** [comment]
Its working now.

` conan remote update conan-center https://center.conan.io`
